lego/apps/companies/serializers.py

```diff
@@ -79,10 +80,21 @@ class CompanyListSerializer(BasisModelSerializer):
         }
     )
 
+    event_count = serializers.SerializerMethodField()
+
+    def get_event_count(self, obj):
+        return obj.events.filter(start_time__gt=timezone.now()).count()
+
+    joblisting_count = serializers.SerializerMethodField()
+
+    def get_joblisting_count(self, obj):
```

Comment:
Have you checked how many `sql`-queries these two method fields generates? I suspect they will create two queries per company, and that would result in `1 + (2 * companyCount)` queries (if all the prefetching is done in the view). :thinking:

Reply:
As far as I can tell, they only generate 1 query each, not sure how to properly check this though.